Runtime/RenderPipeline/PSXRenderPipeline.cs

```diff
@@ -1643,13 +1647,7 @@ static void DrawOpaque(ScriptableRenderContext context, Camera camera, RenderQue
                 perObjectData = ComputePerObjectDataFromLightingVolume(camera)
             };
             
-            var filteringSettings = new FilteringSettings()
-            {
-                renderQueueRange = range,
-                layerMask = camera.cullingMask, // Respect the culling mask specified on the camera so that users can selectively omit specific layers from rendering to this camera.
-                renderingLayerMask = UInt32.MaxValue, // Everything
-                excludeMotionVectorObjects = false
-            };
+            var filteringSettings = new FilteringSettings(range, camera.cullingMask);
```

Comment:
Using the default constructor doesn't properly initialize all fields, which leads to objects not rendering. This constructor uses sensible defaults (including the ones that were being used by HPSXRP's existing code).